app/models/api/area.rb

```diff
@@ -7,6 +7,11 @@ class Scaling < ActiveResource::Base
     self.site = Api::Area.site
   end
 
+  class Init < ActiveResource::Base
```

Comment:
> Why are the initializer inputs sent with the area?

Eh,, good question. I think they can be removed [here](https://github.com/quintel/etengine/blob/master/app/controllers/api/v3/areas_controller.rb) I guess. See this code as a patchy way of fixing the cookie bug.